### PR TITLE
Add optional language mapping

### PR DESCRIPTION
**Pull request by PlutoRane**
- Added optional language mapping

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "notion-translations",
   "main": "build/index.js",
-  "version": "1.5.1",
+  "version": "1.6.0",
```
Comment (KaHLK) [inline package.json]: I'd prefer do bump this in a separate commit 😄

File: src/commands/gen.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { mkdir } from "fs/promises";
 import { get_from_database } from "../api";
 import { GenCache } from "../cache";
 import { Config } from "../config";
-import { Database } from "../model";
+import { Database, LangMapping } from "../model";
```
Comment (KaHLK) [inline src/commands/gen.ts]: I don't think this import is required.

**Review by KaHLK — APPROVED** [review]
Looks good 😄 
I'll quickly fix the issues myself, and then I'll merge.

File: package.json (hunk 1 above)
Comment (PlutoRane) [reply package.json]: Noted! :)
